msprime/trees.py

```diff
@@ -697,7 +697,7 @@ def population(self, u):
 
     def get_population(self, u):
         """
-        Returns the population associated with the specified node. For leaf
+        Returns the population associated with the specified node. For sample
```

Comment:
I am not sure this distinction needs to be made?  If not, could delete this sentence.